modulefiles/tasks/cheyenne/get_obs.local.lua

```diff
@@ -1 +1 @@
-load("miniconda_regional_workflow")
+load("miniconda_regional_workflow-cmaq")
```

Comment:
@padhrigmccarthy, can you remove this change? Online-CMAQ doesn't run the 'get_obs' task.

Reply:
Oops. Failed to push this file. Resolved now.